logic/ippool.go

```diff
@@ -0,0 +1,300 @@
+package logic
+
+import (
+	"container/heap"
+	"errors"
+	"net"
+	"net/netip"
+	"sync"
+
+	"github.com/c-robinson/iplib"
+	"github.com/gravitl/netmaker/database"
+	"github.com/gravitl/netmaker/logger"
+	"github.com/gravitl/netmaker/models"
+	"golang.org/x/exp/slog"
+)
+
+var (
+	ipPool      map[string]PoolMap
```

Comment:
shouldn't this be intialised here `make(map[string]PoolMap)` ?

Reply:
done